litex/build/xilinx/ise.py

```diff
@@ -114,7 +122,10 @@ def _run_ise(build_name, ise_path, source, mode, ngdbuild_opt,
         settings = common.settings(ise_path, ver, "ISE_DS")
         build_script_contents += source_cmd + tools.cygpath(settings) + "\n"
     if mode == "edif":
-        ext = "edif"
+        ext = "ngo"
```

Comment:
Why is this needed now? I'm a little worried it will break other edif use cases?

Reply:
ngo and ngc are supported by ngdbuild and other commands in flow. Adding edif2ngd is cleaner way to do the flow instead of using edif directly (Think for actually did not work fine at all for me , but away right now from developer machine, so can confirm tomorrow)

Reply:
Just tested. Was right, this was broken before. Without change it fails at this line :

netgen -ofmt verilog -w -sim top.edif top_synth.v

Release 14.7 - netgen P.20131013 (lin64)
Copyright (c) 1995-2013 Xilinx, Inc.  All rights reserved.
ERROR:NetListWriters:286 - Invalid file argument 'top.edif'.

Thing is netgen works with ngc or ngo, and edif was sent, that is why I add step to convert edif to ngc and made this work actually.

Reply:
Great!